tests/data/configs/hawq/inception_v3_cifar10_mixed_int.json

```diff
@@ -1,6 +1,5 @@
 {
     "model": "inception_v3",
-    "dataset": "CIFAR10",
```

Comment:
please remove this section, because it's handled in tests now
```
"batchnorm_adaptation": {
    "num_bn_adaptation_samples": 2,
    "num_bn_forget_samples": 1
}
```